kubemarine/procedures/check_iaas.py

```diff
@@ -863,7 +863,7 @@ def install_tcp_listener(cluster: KubernetesCluster, nodes: dict, tcp_ports):
 def check_tcp_ports(cluster):
     with TestCase(cluster.context['testsuite'], '011', 'Network', 'TCPPorts', default_results='Connected'),\
             suspend_firewalld(cluster):
-        tcp_ports = ["80", "443", "6443", "2379", "2380", "10250", "10251", "10252", "30001", "30002"]
+        tcp_ports = ["80", "443", "179", "6443", "8443", "2379", "2380", "10250", "10257", "10259", "30001", "30002"]
```

Comment:
Shouldn't Prometheus port (10254) be also added here?

Reply:
As we are not installing monitoring components (Prometheus, Grafana, etc) by default with k8s, TCP port 10254 is not added in mandatory checklist.